cli/src/features/unit-test-configs/config-case-3.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Can we add a description for each config case to explain the specifics of each case ?

Reply:
Hmmm, I'm of the opinion that the files speak for themselves. Later down the line, these files are expanded and used in multiple test cases, so to write up a description for each one is not going to be so straight forward, because they are used so broadly.